index.js

```diff
@@ -28,6 +28,25 @@ app.post('/add/:quizName/:count', urlencodedParser, async (req, res) => {
     res.redirect()
 })
 
+app.post('/take/:quizName/:userName/score', urlencodedParser, async (req, res) => {
```

Comment:
The /add routes should be saving the questions and expected answers - the key names in redis need to match. See these lines 32 and 33 for what it looks like.